assets/src/edit-story/output/page.js

```diff
@@ -29,27 +29,38 @@ function OutputPage({ page }) {
     height: `calc(100 * ${PAGE_HEIGHT / PAGE_WIDTH} * var(--story-page-vw))`, // 16/9 * 100vw
     maxHeight: `calc(100 * var(--story-page-vh))`, // 100vh
     maxWidth: `calc(100 * ${PAGE_WIDTH / PAGE_HEIGHT} * var(--story-page-vh))`, // 9/16 * 100vh
-    // todo@: this expression uses CSS `min()`, which is still very sparsely
-    // supported.
+    // todo@: this expression uses CSS `min()`, which is still very sparsely supported.
     fontSize: `calc(100 * min(var(--story-page-vh), var(--story-page-vw) * ${PAGE_HEIGHT /
       PAGE_WIDTH}))`,
   };
   const backgroundElements = page.elements.filter(
-    (element) => element.id === page.backgroundElementId
+    (element) =>
+      element.id === page.backgroundElementId &&
+      element.isFullbleedBackground === false
   );
-  const nonBackgroundElements = page.elements.filter(
+  const backgroundFullbleedElements = page.elements.filter(
+    (element) =>
+      element.id === page.backgroundElementId &&
+      element.isFullbleedBackground !== false
+  );
+  const nonBackgroundFullbleedElements = page.elements.filter(
```

Comment:
These three names are a bit confusing, could we perhaps rename `backgroundElements` to `backgroundUnformattedElements` or `backgroundNotFullBleedElements` (or somethibg else)?

To make it clear that `backgroundElements` does not include `backgroundFullBleefElements`.

Also, isn't `nonBackgroundFullbleedElements` just "normal" elements that aren't background? Could we rename this to something less complicated as well? Or keep `nonBackgroundElements`?

Reply:
I know, super confusing. I'll have to change those indeed.

Reply:
Improved a bit now in 756405c92d1ac9a2e715375edf63db00e4b655ff